### DISCUSSION TITLE
Using pretained model in our code

### DISCUSSION THREAD

**Discussion by sart-leo** [MIC-DKFZ/nnUNet#712] [Q&A] [answered]
Hi, I was wondering how to easily use a pretained model (Task005_Prostate) in the code in an easy way. I would like to extract some feature maps of intermediate layers and reuse the latent representation of the bottleneck of one of UNet. I am not very familiar with the framework and you could be of great help.

**Comment by FabianIsensee** [comment, marked as answer]
You should just follow the code of nnUNet_predict. You can replicate it in your script and then access the model. In order to gett access to the feature maps you will need to modify th Generic_UNet
